resources/sass/noscript.scss

```diff
@@ -0,0 +1,65 @@
+@import '~@wikimedia/codex/dist/codex.style.css';
```

Comment:
Do you know why we have to import from `/dist/`? I saw we do it for the app css too but it's not how the [codex docs](https://doc.wikimedia.org/codex/main/using-codex/usage.html#using-design-tokens) advise to do it